src/FunnelChart/FunnelSeries/FunnelAxis/FunnelAxisLabel.tsx

```diff
@@ -48,6 +48,21 @@ export interface FunnelAxisLabelProps {
    * yScale for the funnel. Set internally by `FunnelAxis`.
    */
   yScale: any;
+
+  /**
+   * Positioning for the labels.
+   */
+  position?: 'top' | 'center' | 'bottom';
+
+  /**
+   * Whether to show the value of the data.
+   */
+  showValue?: boolean;
```

Comment:
Is there a case where you REALLY don't want to show the value? Seems pretty important ;P

Reply:
I like to add the default value in this jsdoc comment too.